### PR TITLE
fix testcase failure

### PR DESCRIPTION
**Pull request by pruan-rht**

### CODE REVIEW

**Comment by pruan-rht** [comment]
@akostadinov @liangxia simple one to unblock the OCP on Power team.... PTAL.  Thanks!

**Comment by akostadinov** [comment]
I wonder how it ever worked with the confirmed option. @chao007 ?
/lgtm

**Comment by pruan-rht** [comment]
@chao007 perhaps the CLI command format changed?  Or the framework rule changed?

File: features/storage/add_update_delete_volume.feature [hunk 1]
```diff
@@ -14,7 +14,7 @@ Feature: Add, update remove volume to rc/dc and --overwrite option
       | name         | mydb                       |
     Then the step should succeed
     And a pod becomes ready with labels:
-      | app=mydb |
+      | deploymentconfig=mydb |
```
Comment (qinpingli) [inline features/storage/add_update_delete_volume.feature:17]: @pruan-rht Do you test this with an 4.4 version oc client?

Comment (pruan-rht) [reply features/storage/add_update_delete_volume.feature:17]: This was a regression against 4.3.0

**Pull request merged**